src/System.AppContext/src/System/AppContext.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Windows.ApplicationModel;
+
+namespace System
+{
+    public static class AppContext
+    {
+        [Flags]
+        private enum SwitchValueState
+        {
+            HasFalseValue = 0x1,
+            HasTrueValue = 0x2,
+            HasLookedForOverride = 0x4, // Not used on .NET Native
+            UnknownValue = 0x8 // Not used on .NET Native
+        }
+
+        private static Dictionary<string, SwitchValueState> s_switchMap = new Dictionary<string, SwitchValueState>();
```

Comment:
This is code that already exists internally and isn't being change. I would prefer we file issues for code changes as we are trying to get most of this out on github without changes to the source while porting.